Src/BootCamp.Chapter/AnswerFormatter.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class AnswerFormatter
+    {
+        public string GetFormattedAnswerForPersonWithBiggestLoss(PersonAndBalance personWithBiggestLoss)
+        {
+            return $"{personWithBiggestLoss.GetName()} lost the most money. {FormatCurrency(personWithBiggestLoss.GetLoss())}.";
+        }
+
+        public string GetFormattedAnswerForHighestBalanceEver(PersonAndBalance[] peopleAndBalances, float highestBalanceEver)
+        {
+            var peopleWithSameBalance = GetPeopleNamesWithSameBalance(peopleAndBalances, highestBalanceEver, "highest");
+            var formatedPeopleNames = FormatPeopleNames(peopleWithSameBalance);
+            return $"{formatedPeopleNames} had the most money ever. ¤{highestBalanceEver}.";
+        }
+        
+        public string GetFormattedAnswerForRichestPerson(PersonAndBalance[] peopleAndBalances, float balance)
+        {
+            var peopleWithSameBalance = GetPeopleNamesWithSameBalance(peopleAndBalances, balance, "current");
+            var formatedPeopleNames = FormatPeopleNames(peopleWithSameBalance);
+
+            var areOrIs = (peopleWithSameBalance.Length > 1) ? "are" : "is";
+            var peopleOrPerson = (peopleWithSameBalance.Length > 1) ? "people" : "person";
+            return $"{formatedPeopleNames} {areOrIs} the richest {peopleOrPerson}. ¤{balance}.";
+        }
+
+        public string GetFormattedAnswerForPoorestPerson(PersonAndBalance[] peopleAndBalances, float balance)
+        {
+            var peopleWithSameBalance = GetPeopleNamesWithSameBalance(peopleAndBalances, balance, "current");
+            var formatedPeopleNames = FormatPeopleNames(peopleWithSameBalance);
+
+            var hasOrHave = (peopleWithSameBalance.Length > 1) ? "have" : "has";
+            return $"{formatedPeopleNames} {hasOrHave} the least money. {FormatCurrency(balance)}.";
+        }
+
+        private string FormatCurrency(float number)
+        {
+            var originalCulture = CultureInfo.CurrentCulture;
+            var culture = CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            var clonedNumbers = (NumberFormatInfo)culture.NumberFormat.Clone();
+            clonedNumbers.CurrencyNegativePattern = 1;
+            var formatedBiggestLoss = string.Format(clonedNumbers, "{0:C0}", number);
+            CultureInfo.CurrentCulture = originalCulture;
+
+            return formatedBiggestLoss;
+        }
+
+        private string[] GetPeopleNamesWithSameBalance(PersonAndBalance[] peopleAndBalances, float balance, string typeOfBalance)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'GetPeopleNamesWithSameBalance' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)